plugins/modules/foreman_hostgroup.py

```diff
@@ -248,6 +303,11 @@ def main():
     name, parent = split_fqn(entity_dict['name'])
     entity_dict['name'] = name
 
+    katello_params = ['content_source', 'lifecycle_environment', 'content_view']
+
+    if 'organization' not in entity_dict and list(set(katello_params) & set(entity_dict.keys())):
```

Comment:
this *could* be `required_by` (#399) if we'd move to 2.8+ only (not urging you to change, just thinking aloud about the future and making GH reference this PR in #399 so that when we implement that we know where :))